WWDC/PlaybackViewModel.swift

```diff
@@ -56,6 +56,9 @@ extension Session {
         return asset
     }
 
+    func isVideoWatched() -> Bool {
```

Comment:
This isn't your fault, but this logic is already implemented in `SessionsTableViewController+SupportingTypesAndExtensions.swift:isWatched`. Arguments about proper organization of extensions aside, I think we should just have one implementation 😊